packages/browser-wallet/src/shared/storage/types.ts

```diff
@@ -17,6 +17,7 @@ export enum ChromeStorageKey {
     IsCreatingCredential = 'IsCreatingCredential',
     HasBeenOnboarded = 'hasBeenOnboarded',
     OnboardingLocation = 'onboardingLocation',
+    RecoveryStatus = 'RecoveryStatus',
```

Comment:
Naming convention is `camelCase`.
```suggestion
    RecoveryStatus = 'recoveryStatus',
```